beacon_node/network/src/beacon_processor/worker/sync_methods.rs

```diff
@@ -513,6 +513,21 @@ impl<T: BeaconChainTypes> Worker<T> {
                     })
                 }
             }
+            ref err @ BlockError::ParentExecutionPayloadInvalid { ref parent_root } => {
+                warn!(
+                    self.log,
+                    "Failed to sync chain built on invalid parent";
+                    "parent_root" => ?parent_root,
+                    "advice" => "check execution node for database corruption",
```

Comment:
I have seen a restart fix this sometimes on some ELs. Might be good to advice restarting the EL as a preliminary step . Maybe

"try restarting execution node, if problem persists, check execution node for database corruption"

Reply:
true! restarting Lighthouse after the EL is fixed can also fix things. I'll suggest both.